src/khepri_fun.erl

```diff
@@ -337,7 +337,26 @@ to_standalone_fun2(Fun, State) ->
         fun_kept ->
             {Fun, State};
         undefined ->
-            to_standalone_fun3(Fun, State)
+            undefined
+    end.
+
+to_standalone_fun2_with_retry(Fun, State) ->
```

Comment:
Could you please move this function before `to_standalone_fun2_locked()` (instead of after)?